linter/whitelists/default.txt

```diff
@@ -124,6 +124,8 @@ OSRM
 Overpass
 Overpass
 Open
+Parallax
+parallax
```

Comment:
These should not be in the default whitelist, as they are ordinary English words.